README.md

```diff
@@ -1,10 +1,26 @@
 # bazel-aarch64-linux-gnu
 This repo provides a Bazel cross toolchain for x86_64/Linux host and aarch64/Linux target
```

Comment:
We should update this sentence to include both the cross and native toolchains, yeah?